load.lisp

```diff
@@ -13,6 +13,7 @@
     (load "gclload1.lsp")
     (when ansi
       (load "gclload2.lsp"))
+    (load "tests:ansi-tests;package-local-nicknames.lisp")
```

Comment:
Put this suite in `/ansi-test/beyond-ansi/` instead - PLNs are not ANSI CL-defined behaviour, they are an extension.

Reply:
did that